README.md

```diff
@@ -4,16 +4,22 @@
 
 ### Path matching
 
-If `RUNINSTALL_MATCH` is undefined, or does not match, then all logic will be skipped.
-The command will be passed through to `/usr/local/bin/${cmd}`.
+- `RUNINSTALL_ENABLE_NO_GIT`: if set to `true` it allows running tool update without any git repository. it will bypass `RUNINSTALL_INCLUDES` and `RUNINSTALL_EXCLUDES`
```

Comment:
```suggestion
- `RUNINSTALL_ENABLE_NO_GIT`: if set to `true`, it allows running tool update without any git repository. it will bypass `RUNINSTALL_INCLUDES` and `RUNINSTALL_EXCLUDES`
```

Reply:
me being Justo